webapp/src/images/icons/changes_requested.jsx

```diff
@@ -2,22 +2,8 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {GitPullRequestIcon} from '@primer/octicons-react';
 
-export default class ChangesRequestedIcon extends React.PureComponent {
-    render() {
-        return (
-            <svg
-                viewBox='0 0 16 15'
-                version='1.1'
-                width='16'
-                height='15'
-                role='img'
-            >
-                <path
-                    fillRule='evenodd'
-                    d='M0 1a1 1 0 011-1h14a1 1 0 011 1v10a1 1 0 01-1 1H7.5L4 15.5V12H1a1 1 0 01-1-1V1zm1 0v10h4v2l2-2h8V1H1zm7.5 3h2v1h-2v2h-1V5h-2V4h2V2h1v2zm2 5h-5V8h5v1z'
-                />
-            </svg>
-        );
-    }
-}
+export default () => (
+    <GitPullRequestIcon size='small'/>
```

Comment:
Is that really the right icon? Could you please post a screenshot?

Reply:
https://primer.style/octicons/git-pull-request-16

Reply:
That doesn't seem similar to the previous icon. Is there another that can be used?

Reply:
What about this one? That look familiar https://primer.style/octicons/git-branch-24

Reply:
That looks reasonable. https://primer.style/octicons/diff-16 is more simplistic. I'm 0/5.

@asaadmahmood What do you think?

Reply:
@hanzei Fine with https://primer.style/octicons/file-diff-24